package/README.md

```diff
@@ -0,0 +1,178 @@
+# Raiden Services
+
+## What is this repository
+
+This repository contains the documentation and configuration necessary to run the Raiden Services. The services consist of the Raiden Monitoring service (MS) and the Raiden Pathfinding service (PFS).
+
+## TODO
+<!-- **Current release:** [2018.12.0](https://github.com/raiden-network/raiden-transport/tree/2018.12.0) -->
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Requirements](#requirements)
+  - [Hardware](#hardware)
+  - [Software](#software)
+- [Installation](#installation)
+- [Upgrades](#upgrades)
+- [Known issues](#known-issues)
+- [Changelog](#changelog)
+
+## Overview
+
+The Raiden Network uses pathfinding and monitoring services to increase usability.
+To ensure reliability, availability and neutrality it is desirable that those servers are
+being operated by multiple independent entities.
+
+Therefore we provide this repository which allows easy setup of these services.
+It uses docker and docker-compose for easy installation and upgrades.
+
+### Used software
+
+- docker
+- docker-compose
+- Traefik
+
+### Structure
+
+
+```
++-------------------+
+|                   |
+|   Raiden clients  |
+|                   |
++---------+---------+
+          |
+==========|==========
+          |
++---------v---------+
+|                   |
+|      Traefik      |
+|                   |
++---------+---------+
+          |
+          +------------------------+
+          |                        |
++---------v---------+    +---------v---------+
+|                   |    |                   |
+|     Pathfinding   |    |    Monitoring     |
+|                   |    |                   |
++---------+---------+    +---------+---------+
+```
+
+
+We use Traefik as a reverse proxy and also utilize it's capability of automatically provisiong
+Let's Encrypt TLS certificates.
```

Comment:
ah the sentence continues here, but LetsEncrypt is again different to the previous ones